DataFormats/Phase2TrackerDigi/src/Hit.cc

```diff
@@ -1,8 +1,26 @@
 #include "../interface/Hit.h"
 
 //Describes a hit (meant for use in 4x4 sensor coordinates) by its row number, column number, and adc value
-Hit::Hit(int row_num, int col_num, int adc_num) {
-	row_ = row_num;
-	col_ = col_num;
-	adc_ = adc_num;
+Hit::Hit(int row, int col, int adc) {
+	row_ = row;
+	col_ = col;
+	adc_ = adc;
+}
+
+//Adds the input value to the adc to a max of 15 - used to combine two hits that are at the same position
+void Hit::addADC(int adc) {
+	adc_ += adc;
+
+	if(adc_ > 15) {
+		adc_ = 15;
+	}
+}
+
+//Adds the input value to the adc to a max of 15 - used to combine two hits that are at the same position
+void Hit::addADC(int adc) {
```

Comment:
Isn't this the same function as on line 11?

Reply:
Yes, it would seem to be. I'm not sure how that got duplicated. Thanks for pointing that out!